### DISCUSSION TITLE
Building the plugin

### DISCUSSION THREAD

**Discussion by joeybagofdonuts** [sgkoishi/PackTracker#23] [Q&A] [answered]
Hi Sgkoishi.

So happy you are keeping up with this plugin.  I was going to try to make a small change and request that it was added, but I'm having trouble getting the solution to build.  

Are there any tips that you can give to get this up and running?

**Comment by sgkoishi** [comment, marked as answer]
You may check the `.github/workflows/build.yml` file for how to build the plugin.

To build, use your IDE or run PowerShell script
https://github.com/sgkoishi/PackTracker/blob/ec51198abea78f9de9a38b345283555c2fd07d7c/.github/workflows/build.yml#L30

To setup, manually download the latest HDT and copy them to `./HDT/` folder, or run PowerShell script
https://github.com/sgkoishi/PackTracker/blob/ec51198abea78f9de9a38b345283555c2fd07d7c/.github/workflows/build.yml#L23-L25
(Generally, you only need to run setup once. However, to avoid incompatibility, it is suggested to delete the `.\HDT\` folder and re-run setup if HDT is updated.)


~~or you can just fork this repo and write online, github actions will build it for you~~
